docs/user/troubleshooting.md

```diff
@@ -99,6 +99,230 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 
 The easiest way to get more debugging information from the installer is to check the log file (`.openshift_install.log`) in the install directory. Regardless of the logging level specified, the installer will write its logs in case they need to be inspected retroactively.
 
+### Installer fails to initialize the cluster
+
+The installer uses the [cluster-version-operator] to create all the components of an OpenShift cluster. When the installer fails to initialize the cluster, the most important information is looking at the [clusterversion] and [clusteroperator] objects:
+
+1. Inspecting the `clusterversion` object.
+
+```console
+$ oc --config=${INSTALL_DIR}/auth/kubeconfig get clusterversion -oyaml
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
+metadata:
+  creationTimestamp: 2019-02-27T22:24:21Z
+  generation: 1
+  name: version
+  resourceVersion: "19927"
+  selfLink: /apis/config.openshift.io/v1/clusterversions/version
+  uid: 6e0f4cf8-3ade-11e9-9034-0a923b47ded4
+spec:
+  channel: stable-4.0
+  clusterID: 5ec312f9-f729-429d-a454-61d4906896ca
+  upstream: https://api.openshift.com/api/upgrades_info/v1/graph
+status:
+  availableUpdates: null
+  conditions:
+  - lastTransitionTime: 2019-02-27T22:50:30Z
+    message: Done applying 4.0.0-0.alpha-2019-02-27-131049
+    status: "True"
+    type: Available
+  - lastTransitionTime: 2019-02-27T22:50:30Z
+    status: "False"
+    type: Failing
+  - lastTransitionTime: 2019-02-27T22:50:30Z
+    message: Cluster version is 4.0.0-0.alpha-2019-02-27-131049
+    status: "False"
+    type: Progressing
+  - lastTransitionTime: 2019-02-27T22:24:31Z
+    message: 'Unable to retrieve available updates: unknown version 4.0.0-0.alpha-2019-02-27-131049'
+    reason: RemoteFailed
+    status: "False"
+    type: RetrievedUpdates
+  desired:
+    image: registry.svc.ci.openshift.org/openshift/origin-release@sha256:91e6f754975963e7db1a9958075eb609ad226968623939d262d1cf45e9dbc39a
+    version: 4.0.0-0.alpha-2019-02-27-131049
+  history:
+  - completionTime: 2019-02-27T22:50:30Z
+    image: registry.svc.ci.openshift.org/openshift/origin-release@sha256:91e6f754975963e7db1a9958075eb609ad226968623939d262d1cf45e9dbc39a
+    startedTime: 2019-02-27T22:24:31Z
+    state: Completed
+    version: 4.0.0-0.alpha-2019-02-27-131049
+  observedGeneration: 1
+  versionHash: Wa7as_ik1qE=
+```
+
+Some of most important [conditions][cluster-operator-conditions] to take note are `Failing`, `Available` and `Progressing`. You can look at the conditions using:
+
+```console
+$ oc--config=${INSTALL_DIR}/auth/kubeconfig get clusterversion -o=jsonpath='{.status.conditions}'
```

Comment:
I think this needs `get clusterversion version` (just `get clusterversion` returns a list):

```
$ oc get -o json clusterversion version | head -n4
{
    "apiVersion": "config.openshift.io/v1",
    "kind": "ClusterVersion",
    "metadata": {
$ oc get -o json clusterversion | head -n4
{
    "apiVersion": "v1",
    "items": [
        {
```

Reply:
Also needs a space between `oc` and `--config`